app/Providers/AppServiceProvider.php

```diff
@@ -17,6 +24,17 @@ public function boot()
         $this->bootEloquentMorphs();
         $this->bootMacros();
         $this->bootHorizon();
+
+        DB::whenQueryingForLongerThan(500, function (Connection $connection, QueryExecuted $event) {
+            Notification::send(
+                new AnonymousNotifiable,
+                new SlowQueryLogged(
+                    $event->sql,
+                    $event->time,
+                    Request::url(),
+                ),
+            );
+        });
```

Comment:
```suggestion
        DB::whenQueryingForLongerThan(500, function (Connection $connection, QueryExecuted $query) {
            Notification::send(
                new AnonymousNotifiable,
                new SlowQueryLogged(
                    $query->sql,
                    $query->time,
                    Request::url(),
                ),
            );
        });
```

Reply:
Actually it is an event: `Illuminate\Database\Events\QueryExecuted`. So `$event` makes more sense here.